Xamarin.Forms.Platform.iOS/VisualElementRenderer.cs

```diff
@@ -150,6 +152,126 @@ public void SetElementSize(Size size)
 
 		public event EventHandler<ElementChangedEventArgs<TElement>> ElementChanged;
 
+		protected int TabIndex { get; set; } = 0;
+
+		protected bool TabStop { get; set; } = true;
+
+		protected void UpdateTabStop () => TabStop = Element.IsTabStop;
+
+		protected void UpdateTabIndex () => TabIndex = Element.TabIndex;
+
+		public NativeView FocusSearch (bool forward)
+		{
+			var allChildrens = Application.Current.WalkChildren<VisualElement> ();
```

Comment:
Is there any way we could consolidate some of the code in the two versions of `FocusSearch`? The code for grouping the elements by tab index and the code for searching foward/back is all the same; for maintenance purposes, it'd be nice to have it all in one or two methods.

(Not a deal breaker if consolidating it is just too awkward.)